Makefile

```diff
@@ -21,10 +21,7 @@ PAPERSIZE    := -D latex_paper_size=a4
 SPHINXFLAGS  := -a -W -n -A local=1 $(PAPERSIZE) -d $(BUILDDIR)/doctree
 SPHINXOPTS   := $(SPHINXFLAGS) $(SOURCE)
 
-ENSURECMD=\
-if [[ $(shell which $(1) > /dev/null 2>&1; echo $$?) -eq 1 ]]; then \
-  echo "*** Make sure that $(1) is installed and on your path" && exit 1; \
-fi
+ENSURECMD=which $(1) || (echo "*** Make sure that $(1) is installed and on your path" && exit 1)
```

Comment:
Nice. But need to suppress additional stdout/stderr output caused by which call.